### ISSUE TITLE
Play at speed slider: Using the keyboard it can be not straightforward to get back to a speed of 1.00

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#4619]
### Steps to reproduce

1. With the play at speed slider being the focus, and the initial speed 1.00.
2. Using either right/left arrow, or page up/down change the speed to either the min of max value, and then try to change it back to 1.00

### Expected behavior

You can easily change the speed back to 1.00

### Actual behavior

The closest you can get is 1.01

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

A work around for this is: Extra > Play-at-speed > Adjust playback speed, where you can set the speed to 1.00.

This is a minor bug, but was reported on the Audacity4Blind mailing list.

**Comment by DavidBailes** [comment]
@LWinterberg Could the review of this fix be moved to 3.4. It's a simple safe fix.

**Comment by crsib** [comment]
Honestly, we've discussed it and ... forgot about it. I have a strong opinion that step should be an integer, later divided by whatever divisor that fits. Floating point magic will always have unexpected corner cases.

**Comment by DavidBailes** [comment]
> Honestly, we've discussed it and ... forgot about it. I have a strong opinion that step should be an integer, later divided by whatever divisor that fits. Floating point magic will always have unexpected corner cases.

I think that the issue is caused by the minimum play speed being 0.01 rather than 0.0. I don't think having integer steps would help.

**Comment by crsib** [comment]
> being 0.01 

I don't think that this value is cut in stone. We can easily go with 1/60 (or 1/600, or whatever) as a minimum, so we have the slider ranger in [1, 120] range

**Comment by crsib** [comment]
We can even have a special treatment of zero, remapping it to 0.01

**Comment by DavidBailes** [comment]
> We can even have a special treatment of zero, remapping it to 0.01

I did think about that when I was writing my fix. From what I remember constants for range and increments are not in the same place, so it wasn't straightforward to write the special treatment that wouldn't break if someone changed one of the constants. I also thought that that my fix was good enough for most users, and certainly better than the existing code.
But if you think that the fix should be done better, I can close my PR.
